crates/krun-sys/Cargo.toml

```diff
@@ -10,6 +10,7 @@ links = "krun"
 
 [build-dependencies]
 bindgen = { version = "0.69.4", default-features = false }
+pkg-config = { version = "0.3" }
```

Comment:
We should be explicit here and not use default features.

Reply:
Done, thanks.